.github/workflows/test.yml

```diff
@@ -0,0 +1,310 @@
+name: SalesforceCommerceCloud/pwa-kit/test
+on:
+  schedule:
+    - cron: 0 8 * * *
+  push:
+     branches:
+     # TODO: Before merging remove branches filter
+     - github-actions
+jobs:
+  pwa-kit:
+    strategy:
+      fail-fast: false
```

Comment:
why do we set this to false? I think fail fast is very useful to have.

Reply:
We need `fail-fast: false` because this allows all queued jobs in the matrix to run even if one fails! Without it, if one job fails, then GitHub will cancel all in-progress and queued jobs